apps/monitoring/sbox/00/botkube-values.enc.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+data:
+  values.yaml: ENC[AES256_GCM,data:jIHY/SEGX0cqvyQ2m7qjEvwolKMEXyIpqW2p0659+xpoDFKGC+TOZ+IfdQbwJ0xLb9NT42/2NRfICiG9RuXFMz3zfvrMOE8wwgW8ZgeSfbkt4/UZqhyFvEd1CXc7OBKsCi9cxAKhMz5BLrwvNh9anSI0KOTrKJNJIU51B8urQ5qd0KPSoVuepYFbs7/e7QHanLy7TQeL0MF8dKHijOm4POMR6kyeyWRHMa9pe1ilDyDwYloNFGVzzBhviOEPk3seGs95DBkHwen4MvDWi/wLUSTs9TYw6hkdAOxZ1p/fEFNtlZzLkmlrYpTknQvi7sI/Wtxh/rYnNU9uB0oFbMswKUg7Qza1p4XRLrx9McdSwUUHd4luAg2N/Hf/CAwBCOV0LEjwlo3hnfUNLeYR/Y67QXHJC62XuBlhQ2SJZQ==,iv:U+xhNAaurVswy/wVmBqxL5/tuYi7uFq/492rt+9wPRQ=,tag:Me+hDsfeKk3G5haWoyOeEQ==,type:str]
+kind: Secret
+metadata:
+  creationTimestamp: null
+  name: botkube-values
+  namespace: monitoring
+type: Opaque
+sops:
+  kms: []
+  gcp_kms: []
+  azure_kv:
+    - vault_url: https://dcdcftappsprodkv.vault.azure.net
```

Comment:
You'll need to re-encrypt this on the sandbox keyvault

Reply:
set up here @endakelly  https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/bf308a5c-0624-4334-8ff8-8dca9fd43783/resourceGroups/cnp-core-infra/providers/Microsoft.KeyVault/vaults/infra-vault-sandbox/keys